### DISCUSSION TITLE
Auto Play video on active slide in vertical direcation

### DISCUSSION THREAD

**Discussion by arun2057** [nolimits4web/swiper#5748] [Q&A]
How I can enable the autoplay feature in the video when it comes to the active viewport.
Swiper direction in vertical.
I have to make features like ticktock and youtube shorts.
video autoplay on the active viewport or in an active slide.
I have to scroll the video in a vertical direction.

Any suggestion?

**Comment by Arhell** [comment]
https://codepen.io/intotheprogram/pen/xxggORX
